src/views/HomeView.vue

```diff
@@ -45,23 +45,28 @@ async function loadData () {
   }
 }
 
-async function eventHandler (data) {
-  if (data.operation === 'Details') { // to accounts app
+async function eventHandler (params, statusCallBack) {
```

Comment:
I'm wondering if it's possible to create separate event handlers for these operations. I think it would be much more explicit to add separate event handlers to the components that are emitting these events.